src/components/StoryCreationComponent.js

```diff
@@ -237,6 +254,14 @@ class StoryCreationComponent extends Component {
           Story length: {this.state.storyTextOnly.length} /
           {this.state.storyMaxLength}
           <br />
+          <br />
+          <Recaptcha
+            sitekey="6Ldb85sUAAAAAKe6zdfI6jMm2SBDTzvmJ8iOP9kV"
```

Comment:
Thats awesome that you got the feature in so early! Just a quick comment. Do we keep the key in our code?
Maybe it should fetch it from somewhere?

Reply:
Wherever I have seen - its in the UI component itself. I am not sure if its a problem.

Reply:
Hey - I spoke to Steven about this as well. As we accepting requests only from required urls - It is safe to use this. So, I think this can be merged.